README.adoc

```diff
@@ -3,164 +3,159 @@
 
 To manually initialize the project:
 
-. Navigate to https://start.spring.io.
+. Navigate to https://start.spring.io[https://start.spring.io^].
 This service pulls in all the dependencies you need for an application and does most of the setup for you.
-. Choose either Gradle or Maven and the language you want to use. This guide assumes that you chose Java.
+. Choose either Gradle or Maven and the language you want to use. This guide assumes that you chose Java and Gradle.
 . Click *Generate*.
-. Download the resulting ZIP file, which is an archive of a web application that is configured with your choices.
+. Download the resulting ZIP file, which is an archive of an application that is configured with your choices.
 
 NOTE: If your IDE has the Spring Initializr integration, you can complete this process from your IDE.
 
-NOTE: You can also fork the project from Github and open it in your IDE or other editor.
-
-=== Adding the `awaitility` Dependency
-
-The tests in
-`complete/src/test/java/com/example/schedulingtasks/ScheduledTasksTest.java`
-require the `awaitility` library.
+NOTE: You can also fork the project from Github and open it in your IDE or other editor. Creating a fork will allow you request changes to this guide through submission of a pull request.
+endif::[]
 
-NOTE: Later versions of the `awaitility` library do not work for this test, so you have to
-specify version 3.1.2.
+== Enable Scheduling
 
-To add the `awaitility` library to Maven, add the following dependency:
+Although scheduled tasks can be embedded in web applications, the simpler approach (shown in this guide) creates a standalone application. To do so, package everything in a single, executable JAR file, driven by a Java main() method. The following snippet (from `src/main/java/com/example/schedulingtasks/SchedulingTasksApplication.java`) shows the application class:
 
 ====
-[source,xml]
+[source,java]
 ----
-<dependency>
-  <groupId>org.awaitility</groupId>
-  <artifactId>awaitility</artifactId>
-  <version>3.1.2</version>
-  <scope>test</scope>
-</dependency>
+@SpringBootApplication
+@EnableScheduling
+public class SchedulingTasksApplication {
 ----
 ====
 
-The following listing shows the finished `pom.xml` file:
+Spring Initializr will add the `@SpringBootApplication` annotation to our main class. `@SpringBootApplication` is a convenience annotation that adds all of the following:
 
-====
-[source,xml]
-----
-include::complete/pom.xml[]
-----
-====
+- `@Configuration`: Tags the class as a source of bean definitions for the application
+context.
+- `@EnableAutoConfiguration`: Spring Boot attempts to automatically configure your Spring application based on the dependencies that you have added.
+- `@ComponentScan`: Tells Spring to look for other components, configurations, and
+services. If specific packages are not defined, scanning will occur recursively beginning with the package of the class that declares the annotation.
```

Comment:
Change `scanning will occur recursively beginning` to `recursive scanning begins`